.github/workflows/ansible-test.yml

```diff
@@ -46,16 +46,16 @@ jobs:
       # .../ansible_collections/${{env.NAMESPACE}}/${{env.COLLECTION_NAME}}/
 
       - name: Check out code
-        uses: actions/checkout@v2
+        uses: actions/checkout@v3
         with:
           path: ansible_collections/${{env.NAMESPACE}}/${{env.COLLECTION_NAME}}
 
       - name: Set up Python
-        uses: actions/setup-python@v2
+        uses: actions/setup-python@v3
         with:
           # it is just required to run that once as "ansible-test sanity" in the docker image
           # will run on all python versions it supports.
-          python-version: 3.8
+          python-version: '3.10'
```

Comment:
Should we have waited to go to 3.10 until Ansible 2.9 and 2.10 are removed?

Reply:
The tests actually run fine with Python 3.10, so bumping it now will allow to keep this value for longer time :)

Reply:
@felixfontein FYI I've recently faced a bug in Ansible 2.10 under Python 3.9+ that makes installing old `community.general` versions error out (the bug is actually in CPython's `tarfile` stdlib module). So while the tests may succeed, it doesn't mean that it's without bugs. With this in mind, the bump may be too rushed after all.

Reply:
If you experience problems for a specific collection you can still downgrade to an earlier Python version. Also 3.8 won't work with `devel` anymore, so not bumping up the version would result in more widespread errors. I guess it would be best to make the Python version depend on the ansible-core version, but that's a lot more tricky... (and a lot easier to solve in a good way with the shared workflow - though right now that lets the Python version default to 3.9, which would also not be OK according to your example.)